src/document/json/strings.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * `EscapeString` escapes string
+ *
+ */
+export function escapeString(string: string): string {
+  return ('' + string).replace(
```

Comment:
The parameter type is a string. 
I think typecasting to string is unnecessary. 

```suggestion
export function escapeString(str: string): string {
  return str.replace(
```

Reply:
Thank you for your suggesstion. I agree that typecasting is unnecessary!